readme.md

```diff
@@ -9,6 +9,7 @@ Multi-platform and multi-language.
 [![codecov](https://codecov.io/gh/renovatebot/renovate/branch/main/graph/badge.svg)](https://codecov.io/gh/renovatebot/renovate)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![Build status](https://github.com/renovatebot/renovate/workflows/build/badge.svg)](https://github.com/renovatebot/renovate/actions)
+![Docker Pulls](https://img.shields.io/docker/pulls/renovate/renovate?color=turquoise)
```

Comment:
Let's zoom out a bit, and figure out what the goal is here.

If we care about the number of Renovate users, then Docker pulls is a misleading number to show. There are _way_ more Docker pulls than users. The badge shows 500 million Docker pulls.

Let's check what's a reasonable number. GitHub says it has 83+ million developers on it. GitLab says it has 30 million registered users. Not all users will run Renovate. So whatever number we show should be less than the total userbase for GitHub and Gitlab. 😄 

A more meaningful number would be the number of Renovate GitHub hosted app users. Those users are only counted once. But showing that number might be a bad business idea, because that tells competitors how many users we have.

Reply:
Let's let the competitors think we have over 500M users ;)

On a more serious note though - I definitely agree that the number of Docker pulls doesn't indicate anything about the number of Renovate users, but this large number is a testament to the scale and popularity Renovate has reached. When comparing to other well-known projects, the number of pulls for Renovate is comparatively high.

Regarding the number of GitHub hosted app users - I believe that number is publicly viewable via the GitHub marketplace.

Reply:
I think 500m Docker pulls is an amazing number to show off and great validation for the project